lib/exact/index.ts

```diff
@@ -87,6 +87,10 @@ type ArrayExact<TValue extends readonly any[], TShape extends readonly any[]> =
 // `isBCorBC3(bcOrBc3)` fails because `bcOrBc3` is inferred as `BC`???
 type PrimitiveExact<TValue, TShape> = [TValue] extends [TShape] ? ([TShape] extends [TValue] ? TValue : never) : never;
 
+
+/**
+ * @deprecated `Exact` will be removed in v11.0. Please use `IsExact`
+ */
 export type Exact<TValue, TShape> = [TValue] extends [readonly any[]]
```

Comment:
You can simplify this file to:

```ts
import { IsExact } from "../is-exact";

/**
 * @deprecated `Exact` will be removed in v11.0. Please use `IsExact`
 */
export type Exact<TValue, TShape> = IsExact<TValue, TShape>;
```

Benefits are:

1. Implementation is the same
2. Reduced repetitiveness
3. Easier to remove in major release